docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc

```diff
@@ -14,6 +14,10 @@ You can use a migration wave to create Jira issues automatically for each applic
 
 NOTE: You cannot delete an application if it is linked to a Jira ticket or is associated with a migration wave. To unlink the application from the Jira ticket, click the `Unlink from Jira` icon in the details view of the application or in the details view of a migration wave.
 
+.Prerequisites
+
+* You configured Jira connection. For more information, see xref:creating-configuring-jira-connection_user-interface-guide[Creating and configuring a Jira connection].
```

Comment:
Chapter 5 of this guide (Creating and configuring a Jira connection) should be a prerequisite for this procedure, is that right? If it's not true, will remove it. Let me know, please :-)

Reply:
You're right!

Reply:
Absolutely, @kpunwatk, will check it when I'll be pushing the next commit! Thanks a lot for reporting this issue :-)